backend-nodejs/expressApp/routes/UploadRouter.js

```diff
@@ -0,0 +1,60 @@
+//todo(donny): clear out upload folder after the s3 bucket is done with it.
+require("dotenv").config();
+const fs = require("fs").promises;
+
+const express = require("express");
+const multer = require("multer");
+
+const AWS = require("aws-sdk");
+const s3 = new AWS.S3({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_KEY,
+});
+
+let storage = multer.diskStorage({
+  destination: function (req, file, cb) {
+    cb(null, "./uploads/");
+  },
+  filename: function (req, file, cb) {
+    cb(null, file.originalname);
+  },
+});
```

Comment:
It is probably ok for now saving to disk first, but it may not be necessary if we can pass the file stream directly to s3 without having to save it to disk first.